test/libgit2-online.jl

```diff
@@ -21,12 +21,18 @@ mktempdir() do dir
                 finalize(repo)
             end
         #end
+        try
         #@testset "with incorrect url" begin
             repo_path = joinpath(dir, "Example2")
-            # credential are required because github try authenticate on uknown repo
-            x = Nullable(LibGit2.UserPasswordCredentials("X","X"))
-            @test_throws LibGit2.Error.GitError LibGit2.clone(https_prefix*repo_url*randstring(10), repo_path, payload=x)
+            # credential are required because github try authenticate on unknown repo
+            cred = LibGit2.UserPasswordCredentials("","") # empty credentials cause authentication error
+            LibGit2.clone(https_prefix*repo_url*randstring(10), repo_path, payload=Nullable(cred))
+            error("unexpected")
         #end
```

Comment:
the commented end-of-testset should presumably be after the catch, not in the middle of it